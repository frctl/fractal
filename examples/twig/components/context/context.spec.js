const { context } = require('@frctl/adapter-tests');

const fractal = require('../../fractal.config.js');

describe('tree', () => {
    beforeEach(async () => {
        await fractal.load();
    });

    context(fractal);
});
