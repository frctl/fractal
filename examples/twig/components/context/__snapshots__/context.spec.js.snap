// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tree asynchronously loads context 1`] = `
Object {
  "xkcd": Object {
    "alt": "Proper User Policy apparently means Simon Says.",
    "day": "28",
    "img": "https://imgs.xkcd.com/comics/sandwich.png",
    "link": "",
    "month": "8",
    "news": "",
    "num": 149,
    "safe_title": "Sandwich",
    "title": "Sandwich",
    "transcript": "[[ A man is sitting on a couch, talking to another man.  They are both stick figures. ]]
First man:  Make me a sandwich.
Second man:  What?  Make it yourself.
First man:  Sudo make me a sandwich.
Second man:  Okay",
    "year": "2006",
  },
}
`;
